src/input_handler.cc

```diff
@@ -298,7 +298,18 @@ class Normal : public InputMode
 
     DisplayLine mode_line() const override
     {
-        AtomList atoms = { { to_string(context().selections().size()) + " sel", get_face("StatusLineInfo") } };
+        AtomList atoms;
+        if (context().selections().size() == 1)
+        {
+            atoms.emplace_back(to_string(context().selections().size()) + " sel", get_face("StatusLineInfo"));
+        }
+        else
+        {
+            atoms.emplace_back(to_string(context().selections().main_index() + 1) + '/'
+                             + to_string(context().selections().size()) + " sels", get_face("StatusLineInfo"));
```

Comment:
`format("{}/{} sels", ...)` although I am not really satisfied with that format (hehe), how about `<count> sels (<main>)`, suggestions ?